Dockerfile

```diff
@@ -1,30 +1,38 @@
-FROM openjdk:11-jre
-
+FROM alpine:latest as builder
```

Comment:
Didn't we had problem with alpine before

Reply:
Yes, but here it's only used as the builder - the real image your Zeebe container will use is `openjdk:11-jre-slim`.

Reply:
See https://docs.docker.com/develop/develop-images/multistage-build/

Reply:
:+1: